src/scripts/hamburger.js

```diff
@@ -0,0 +1,22 @@
+function burgerNavigation() {
+  const mobileIcon = document.getElementById("mobile-navigation");
+  const mobileBars = mobileIcon.querySelector(".nav-bars");
+  const navigation = document.getElementById("navigation-list");
+  const allLinks = navigation.querySelectorAll("li");
```

Comment:
it's not links

Reply:
Should be: allListItem?

Reply:
`listItems` would be enough. We don't define any other listItems in the script